packages/react/src/views/ChatInput/ChatInputFormattingToolbar.js

```diff
@@ -42,7 +43,9 @@ const ChatInputFormattingToolbar = ({
 
   const handleEmojiClick = (emojiEvent) => {
     const [emoji] = emojiEvent.names;
-    messageRef.current.value += ` :${emoji.replace(/[\s-]+/g, '_')}: `;
+    const newValue = messageRef.current.value + ` :${emoji.replace(/[\s-]+/g, '_')}: `;
```

Comment:
Thanks for your contribution, @smritidoneria.
LGTM! Can you please give a meaningful name to this variable instead of `newValue`?

Reply:
Yes,sure